src/tiledb/cloud/soma/ingest.py

```diff
@@ -168,8 +175,12 @@ def run_ingest_workflow_udf(
             entry_output_uri += base
             logger.debug("ENUMERATOR ENTRY_OUTPUT_URI=%r", entry_output_uri)
 
-            if pattern is not None and not re.match(pattern, entry_input_uri):
-                logger.debug("ENUMERATOR SKIP NO MATCH ON <<%r>>", pattern)
+            if pattern is not None and not re.search(pattern, entry_input_uri):
+                logger.info(
+                    "Skipping non-matching input: pattern=%r, entry_input_uri=%r",
```

Comment:
I'm going to start nudging all log entry into this form. `Event happened: var1=foo, var2=bar`.